submitted_models/marble_husky_sensor_config_1/launch/spawner.rb

```diff
@@ -0,0 +1,72 @@
+def spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw)
+  <<-HEREDOC
+  <spawn name='#{_name}'>
+    <name>#{_name}</name>
+    <allow_renaming>false</allow_renaming>
+    <pose>#{_x} #{_y} #{_z + 0.2} #{_roll} #{_pitch} #{_yaw}</pose>
+    <world>#{_worldName}</world>
+    <is_performer>true</is_performer>
+    <sdf version='1.6'>
+    <include>
+      <name>#{_name}</name>
+      <uri>#{_modelURI}</uri>
+      <!-- Diff drive -->
+      <plugin filename="libignition-gazebo-diff-drive-system.so"
+              name="ignition::gazebo::systems::DiffDrive">
+        <left_joint>front_left_wheel_joint</left_joint>
+        <left_joint>rear_left_wheel_joint</left_joint>
+        <right_joint>front_right_wheel_joint</right_joint>
+        <right_joint>rear_right_wheel_joint</right_joint>
+        <wheel_separation>#{0.45649 * 1.5}</wheel_separation>
+        <wheel_radius>0.1651</wheel_radius>
+        <topic>/model/#{_name}/cmd_vel_relay</topic>
```

Comment:
Please, consider adding this block to limit the maximum velocities and accelerations:

```
"      <min_velocity>-1</min_velocity>\n"\
"      <max_velocity>1</max_velocity>\n"\
"      <min_acceleration>-2</min_acceleration>\n"\
"      <max_acceleration>3</max_acceleration>\n"\
```

These parameters have been introduced [here](https://github.com/ignitionrobotics/ign-gazebo/pull/161). You should also add it to the same file under `robotica_x2_sensor_config_1`.

Reply:
Done. PR will be submitted shortly to reflect these changes.

Reply:
Sorry, I was reusing the same comment from other team. Feel free to ignore my comment about `robotika`.